src/test/ui/borrowck/borrowck-if-with-else.stderr

```diff
@@ -1,8 +1,13 @@
-error[E0381]: use of possibly-uninitialized variable: `x`
+error[E0381]: used binding `x` isn't initialized in all conditions
   --> $DIR/borrowck-if-with-else.rs:10:9
    |
+LL |     let x: isize;
+   |         - binding declared here but left uninitialized
+LL |     if 1 > 2 {
+   |        ----- `x` is uninitialized if this condition is met
```

Comment:
yea, I would expect this message (or its inverse) no matter whether there's an else arm or not.

Reply:
Why? Because the condition's value can be statically determined?

I think what you're saying is that in the case where

```
    if y {
        x = 10;
    } else {
        println!("whoops");
    }
```

The same message can be shown as for

```
    if y {
        x = 10;
    }
```

But that is not the case because for the former we point at the ` else `, which we don't have, that's why that case is special cased.